test/integration/test-fixtures/podman/containers.conf

```diff
@@ -0,0 +1,20 @@
+[containers]
+  log_size_max = -1
+  pids_limit = 2048
+  userns_size = 65536
+
+[engine]
+  num_locks = 2048
+  active_service = "ephemeral"
+  stop_timeout = 10
+  [engine.service_destinations]
+    [engine.service_destinations.ephemeral]
+      uri = "ssh://root@localhost:2222/run/podman/podman.sock"
+# TODO: dont hardcode this... this should be the volume mount from a running container
```

Comment:
if this fixture still used? I think I added this for reference but it isn't used anywhere (so should be deleted)